src/closures/constant_smagorinsky.jl

```diff
@@ -0,0 +1,89 @@
+Base.@kwdef struct ConstantSmagorinsky{T} <: ScalarDiffusivity{T}
+               C :: T = 0.23
```

Comment:
Would `C_s` (or `C_smag`) be more descriptive?

Reply:
The syntax one will use is `smag = ConstantSmagorinsky(C=1.7)`. Then the field is accessed via `smag.C`. 

It's similar to `Clima.ClimaAtmos`, or `G.Gu`. Fields of a struct are shipped with their labels inextricably bound. I am philosophically opposed to adding redundant labels.

Reply:
Ah right ignore this comment then.